app/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "private": true,
+  "engines": {
+    "node": ">=16.14.2",
+    "npm": ">=8.5.0"
+  },
+  "scripts": {
+    "dev": "nuxi dev",
+    "build": "nuxi build",
+    "start": "NODE_ENV=production node .output/server/index.mjs",
+    "lint": "npm-run-all lint:*",
+    "lint:types": "nuxi typecheck",
+    "lint:style": "eslint .",
+    "test": "NODE_ENV=test npm-run-all postinstall && TZ=UTC vitest --run",
+    "test:watch": "npm-run-all postinstall && TZ=UTC vitest",
+    "test:ui": "npm-run-all postinstall && TZ=UTC vitest --ui",
+    "prepare": "cd .. && husky install",
+    "postinstall": "nuxi prepare",
+    "story": "histoire dev",
+    "story:build": "histoire build",
+    "story:preview": "histoire preview"
+  },
+  "dependencies": {
+    "@ant-design/icons-vue": "^6.1.0",
+    "@sidestream-tech/nuxt-sidebase-parse": "^0.2.4",
+    "@vueuse/core": "^9.2.0",
+    "sqlite3": "^5.0.11",
+    "typeorm": "^0.3.9",
+    "zod": "^3.18.0"
```

Comment:
this should not be needed anymore, you can import `zod` from `nuxt-sidebase-parse`!